test/models/report_test.rb

```diff
@@ -3,7 +3,60 @@
 require 'test_helper'
 
 class ReportTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  setup do
+    @user = users(:alice)
+    @report = reports(:alice_report)
+    @mentioned_report = reports(:bob_report)
+  end
+  test 'editable?は@userが日報作成者ならtrueを返す' do
+    assert @report.editable?(@user)
+  end
+
+  test 'editable?は@userが日報作成者じゃないならfalseを返す' do
+    incorrect_user = users(:bob)
+    assert_not @report.editable?(incorrect_user)
+  end
+
+  test 'created_onはDateクラスを返す' do
+    @report.created_at = Time.zone.now
+    assert_instance_of Date, @report.created_on
```

Comment:
```suggestion
    assert_equal Date.today, @report.created_on
```

というように、データ型の確認だけでなく、値の確認までやった方がテストとして安心です。
（Date型だけど昨日の日付なら意味がない）

Reply:
修正しました！
ただ、こちら`Date.today`で修正すると
`Do not use date.today without zone. use time.zone.today instead`とrubocopに怒られてしまいました。
Railsにおける「Timezone情報を持っていないので危険だ」という指摘であることを確認しました。
しかし、`created_on`メソッド自体が日付表示の表現を変えるためだけのものですし、この`assert_equal`部分もテストデータなので、`Date.today`で問題ないのではと考えています。
こういう場合は、rubocopの指摘はそのままでもいいのでしょうか。
それとも、表示が出ないようにしたり、コメントなどで注釈したりする必要がありますか。